### PR TITLE
[skip ci] Updating the test steps for NSX-v setup

### PR DESCRIPTION
**Pull request by rajanashok**
Adding test setup steps to create the NSX-v environment.

### CODE REVIEW

File: tests/manual-test-cases/Group5-Functional-Tests/5-7-NSX.md [hunk 1]
```diff
@@ -11,9 +11,64 @@ To verify the VIC appliance works when the vCenter is using NSX networking
 This test requires access to VMWare Nimbus cluster for dynamic ESXi and vCenter creation
 
 #Test Steps:
-1. Deploy a new vCenter in Nimbus with NSX configured  
-2. Deploy VCH Appliance to the new vCenter
-3. Run a variety of docker commands on the VCH appliance
+1. Deploy NSX
+	/mts/git/bin/nimbus-vsmdeploy --nics 1 --vsmBuild ob-5007049 vic-3-nsxv-mgr
```
Comment (mhagen-vmware) [inline tests/manual-test-cases/Group5-Functional-Tests/5-7-NSX.md]: this isn't providing the formatting that I think you want, might want to try back tics.

File: tests/manual-test-cases/Group5-Functional-Tests/5-7-NSX.md [hunk 2]
```diff
@@ -11,9 +11,64 @@ To verify the VIC appliance works when the vCenter is using NSX networking
...
+
+2. Deploy VC + VSAN
+	/mts/git/bin/nimbus-testbeddeploy --noSupportBundles --vcvaBuild 4944578 --esxPxeDir 4887370 --esxBuild 4887370 --testbedName vcqa-vsan-simple-pxeBoot-vcva --runName vsan
+
+3. Register NSX-v to VC from Manage vCenter Registration, to add the VC info (NSX Credential: admin/default).
+
+4. Assign NSXv License in VC from Assets → Solutions → All Actions → Assign License → Add License → Select the new added license → OK
+   a. Copy one valid NSX license out from https://buildweb.eng.vmware.com/serials/NSX/6.2.1.lic.txt. 
+      One workable license:R44LM-DW35M-N83D1-0U9K2-38THL
```
Comment (mhagen-vmware) [inline tests/manual-test-cases/Group5-Functional-Tests/5-7-NSX.md]: You cannot leak the license here.

Comment (mhagen-vmware) [inline tests/manual-test-cases/Group5-Functional-Tests/5-7-NSX.md]: nor the internal url

Comment (rajanashok) [inline tests/manual-test-cases/Group5-Functional-Tests/5-7-NSX.md]: Sorry about that, will get it removed now.

File: tests/manual-test-cases/Group5-Functional-Tests/5-7-NSX.md [hunk 3]
```diff
@@ -11,9 +11,64 @@ To verify the VIC appliance works when the vCenter is using NSX networking
...
+
+5. Create a Distributed Switch
+   a. Edit the new added DVS MTU value of DVS from default 1500 to 9000
+   b. Select the new added DVS and Add all hosts in the cluster
+   c. Assign vmnic1 for this switch per host (remain vmnic0 with vSwitch0 to get the external IP)
+
+6. Prepare IP of NSX Controller
+   a. Select one host which will create NSX controller in.
+   b. Login the host via ssh, get host's network info 1). Netmask  2). Gateway. 
+      esxcli network ip interface ipv4 get
```
Comment (mhagen-vmware) [inline tests/manual-test-cases/Group5-Functional-Tests/5-7-NSX.md]: back tics here too

File: tests/manual-test-cases/Group5-Functional-Tests/5-7-NSX.md [hunk 4]
```diff
@@ -11,9 +11,64 @@ To verify the VIC appliance works when the vCenter is using NSX networking
...
+      net-vdl2 -l
+   g. Check vmknic created by VXLAN among all hosts are ping-able
+      esxcli network ip interface ipv4 get
+
+10. Create Logical Switch
+    Go to Home → Networking & Security → Installation → Logical Network Preparation
+    Remain info in 'VXLAN Transport' as default
+    Go to 'Segment ID' → Edit → Add the value of 'Segment ID pool', for example '5001-50000'
+    Go to 'Transport Zones' → Click Add → Create a Transport Zone → Add Name / Description properly, 
+    Remain Relication mode as 'Unicast', Select   cluster that will be part of this Transport Zones
```
Comment (mhagen-vmware) [inline tests/manual-test-cases/Group5-Functional-Tests/5-7-NSX.md]: a. b. c. etc?

File: tests/manual-test-cases/Group5-Functional-Tests/5-7-NSX.md [hunk 5]
```diff
@@ -11,9 +11,64 @@ To verify the VIC appliance works when the vCenter is using NSX networking
...
+8. Install NSX components on ESX from Home, Networking & Security, Installation, Host Preparation to all the host in cluster.
+
+9. Configure VXLAN
+   a. Click <Not Configured> in VXLAN column → Keep value as default except 'VMKNic IP Addressing'
+      Default MTU is 1600, make sure the MTU of DVS is larger than this one (set as 9000 manually previously)
+   b. For 'VMKNic IP Addressing' field, default is 'Use DHCP', change it to 'Use IP Pool'.
+   c. Choose 'New IP Pool', add new ip pool info.
+      It's recommended to define an internal IP pool in order to isolate from management network
+      Example: Name → internel-vxlan-ip-pool, Gateway → 192.168.0.254, Prefix Legth → 24, 
+                Static IP Pool → 192.168.0.1-192.168.0.10 (Depends on how many host this VXLAN managed, length of IP pool > num of hosts)
```
Comment (mhagen-vmware) [inline tests/manual-test-cases/Group5-Functional-Tests/5-7-NSX.md]: back tics

File: tests/manual-test-cases/Group5-Functional-Tests/5-7-NSX.md [hunk 6]
```diff
@@ -11,9 +11,64 @@ To verify the VIC appliance works when the vCenter is using NSX networking
...
+   d. Check VXLAN is installed on ESX (vmk1 is newly created)
+   e. Show the vmknic info of VXLAN from Web Client: ( Installation → Logical Network Preparation → VXLAN Transport)
+   f. Check the network settings from the host 
+      esxcli network ip interface list
+      net-vdl2 -l
```
Comment (mhagen-vmware) [inline tests/manual-test-cases/Group5-Functional-Tests/5-7-NSX.md]: back tics

File: tests/manual-test-cases/Group5-Functional-Tests/5-7-NSX.md [hunk 7]
```diff
@@ -11,12 +11,66 @@ To verify the VIC appliance works when the vCenter is using NSX networking
...
+      - `net-vdl2 -l`
+   - Check vmknic created by VXLAN among all hosts are ping-able
+      - `esxcli network ip interface ipv4 get`
+
+10. Create Logical Switch
+    - Go to Home → Networking & Security → Installation → Logical Network Preparation
+    - Remain info in 'VXLAN Transport' as default
+    - Go to 'Segment ID' → Edit → Add the value of 'Segment ID pool', for example '5001-50000'
+    - Go to 'Transport Zones' → Click Add → Create a Transport Zone → Add Name / Description properly, 
+       - `Remain Relication mode as 'Unicast', Select   cluster that will be part of this Transport Zones`
```
Comment (mhagen-vmware) [inline tests/manual-test-cases/Group5-Functional-Tests/5-7-NSX.md]: replication or relocation?

Comment (rajanashok) [inline tests/manual-test-cases/Group5-Functional-Tests/5-7-NSX.md]: copy paste error :)

Comment (rajanashok) [inline tests/manual-test-cases/Group5-Functional-Tests/5-7-NSX.md]: Updated to replication

**Pull request merged**